packages/compass-e2e-tests/tests/smoke.test.ts

```diff
@@ -45,7 +45,7 @@ describe('Smoke tests', function () {
     await afterTest(compass, this.currentTest);
   });
 
-  describe('Sidebar', function () {
```

Comment:
Since this file is getting more tests and as you mentioned more will be coming, should we break it into multiple files? How could that look? Maybe each of these suites gets a file in the smoke tests folder?

Reply:
I was just thinking about that because making a bunch of small PRs against the same file that are in review at the same time makes for annoying merges.

I should probably make each describe its own file as that's the most obvious way forward. The downside is that we'll be re-launching the app and connecting again every time so things will be a bit slower. Probably not the end of the world.

Reply:
I have split it up now.